### PR TITLE
Work on Todo page

### PR DESCRIPTION
**Pull request by ofagbemi**
Update last task, quick complete, and task creation to match mocks. Mostly implement task completion and deletion — I still need to add the icons over the green and red backgrounds that appear when the user swipes.

Remaining todos are
- Add completion / deletion icons
- Use correct photos in last task and kudos button
- Allow users to send kudos

### CODE REVIEW

File: ios/rüm/TodoCell.swift [hunk 1]
```diff
@@ -0,0 +1,143 @@
+//
+//  TodoCell.swift
+//  rüm
+//
+//  Created by Jare Fagbemi on 5/10/16.
+//  Copyright © 2016 Fang Industries. All rights reserved.
+//
+
+import UIKit
+import Spring
```
Comment (andrewfang) [inline ios/rüm/TodoCell.swift]: Where does this come from? Did you check in the Module?

Comment (ofagbemi) [inline ios/rüm/TodoCell.swift]: Whoops, I either forgot to delete that line or forgot to push again. Will do in a sec

Comment (ofagbemi) [inline ios/rüm/TodoCell.swift]: Fixed

**Comment by andrewfang** [comment]
LGTM. Merge when ready!

File: ios/rüm/LastTaskCell.swift [hunk 2]
```diff
@@ -33,18 +33,10 @@ class LastTaskCell: UITableViewCell {
         taskLabel.text = task
         if let imageUrl = NSURL(string: photo) {
             if let data = NSData(contentsOfURL: imageUrl) {
-                self.profileImageView.image = UIImage(data: data)
+                let image = UIImage(data: data)
+                self.profileImageView.image = image
+                self.backgroundImageView.image = image
```
Comment (andrewfang) [inline ios/rüm/LastTaskCell.swift:38]: wait why are we loading the user's profile image into both the background and the cell? I liked it the other way with the background being the task, foreground imageview being the user

**Pull request merged**